README.md

```diff
@@ -44,6 +44,7 @@ Made with ❤️ by [XMARTLABS](http://xmartlabs.com). This is the re-creation o
   + [Multivalued sections]
   + [Validations]
   + [Swipe Actions]
+* [Custom form size]
```

Comment:
I would put this under `Usage`. I think it is not a main item in the Readme like `Usage` or `Custom Rows`.

Reply:
OK, thanks!